src/app/app.html

```diff
@@ -24,7 +24,7 @@
         <div class="header-actions">
             <clr-dropdown [clrMenuPosition]="'bottom-right'">
                 <button class="nav-icon" clrDropdownToggle>
-                    <clr-icon shape="cog"></clr-icon>
+                    <clr-icon shape="settings"></clr-icon>
```

Comment:
I agree with @jaffoneh. I had this discussion with Yen and Lilia earlier this week and we decided to go with more generic names. We should also look into creating aliases & categories for icons. May the cog icons goes in the Settings category or has a Settings alias? This will allow for users to search cog and other icons that can be used for Settings.